<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdjustmentsWindow</class>
 <widget class="QMainWindow" name="AdjustmentsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>647</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>793</width>
    <height>647</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>793</width>
    <height>647</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnOpenBitmap">
        <property name="text">
         <string>Open Bitmap</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveBitmap">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save Bitmap</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblBrigtness">
        <property name="text">
         <string>Brightness:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSlider" name="sldrBrightness">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>-128</number>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="lblBrightnessValue">
          <property name="text">
           <string>+0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lblContrast">
        <property name="text">
         <string>Contrast:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSlider" name="sldrContrast">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>256</number>
          </property>
          <property name="value">
           <number>128</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="lblContrastValue">
          <property name="text">
           <string>+0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QGraphicsView" name="gfxBitmap"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
